### ISSUE TITLE
Visual Studio error: Cannot open source file

### ISSUE THREAD

**Issue by SizzlingCalamari** [dcourtois/premake-compilationunit#4]
Commit 5d9fe726 added the project name into the path for the cpp files.
I'm running into the errors below with this change on premake 5 alpha 11.
Reverting 5d9fe726 fixes the compilation.
Here's the premake file that I'm using: https://github.com/SizzlingStats/demboyz/blob/master/premake/premake5.lua

I'm wondering which version of the alpha and what configuration was being used that needed the project name to be added to the path. I tested alpha versions 6 to 11 and didn't experience any problem without the project name in the path. @tdev ?

```
1>__compilation_unit__1.cpp
1>__compilation_unit__2.cpp
1>__compilation_unit__3.cpp
1>__compilation_unit__4.cpp
1>__compilation_unit__5.cpp
1>__compilation_unit__6.cpp
1>__compilation_unit__7.cpp
1>__compilation_unit__8.cpp
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__1.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__5.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__3.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__4.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__2.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__7.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__8.cpp': No such file or directory
1>c1xx : fatal error C1083: Cannot open source file: 'obj\x32\Debug\demboyz\__compilation_unit__6.cpp': No such file or directory
```

**Comment by dcourtois** [comment]
Hi, thanks for reporting this issue ! And it should be fixed by commit a62371da793cbe793b1527176de7f2151a27ce0d
Let me know if you still have problems :)

**Comment by SizzlingCalamari** [comment]
Confirmed fixed. 👍
